SecurityPkg/SecurityPkg.dec

```diff
@@ -664,6 +676,14 @@
   ## This PCD records LASA field in CC EVENTLOG ACPI table.
   gEfiSecurityPkgTokenSpaceGuid.PcdCcEventlogAcpiTableLasa|0|UINT64|0x00010026
 
+  ## MU_CHANGE: Add a PCD to determine partition ID of TPM Service.
+  #
+  # Define the partition ID for TPM Service.
+  # Default PcdTpmServiceFfaPartitionId = 0x0, which should never be set to by
+  # the FF-A framework.
+  #
+  gEfiSecurityPkgTokenSpaceGuid.PcdTpmServiceFfaPartitionId|0|UINT16|0x0001002B

```

Comment:
Any reason this is fixed vs just discovering the ID at runtime?

Reply:
There was an ask to support multiple TPM services on the secure side... This is added so that if this is 0, we will do discovering on the fly, otherwise we will just stick to this "selected" partition ID to talk to.